src/index.tsx

```diff
@@ -1,12 +1,14 @@
 import React from "react";
 import ReactDOM from "react-dom";
+import { Provider } from "react-redux";
+
 import "./index.css";
 import App from "./App";
-import * as serviceWorker from "./serviceWorker";
```

Comment:
What was this doing?

Reply:
No idea, beyond the comment. I figure we could add it back if we needed it.